src/vs/workbench/browser/layout.ts

```diff
@@ -671,7 +671,7 @@ export abstract class Layout extends Disposable implements IWorkbenchLayoutServi
 
 			// Only restore last viewlet if window was reloaded or we are in development mode
 			let viewContainerToRestore: string | undefined;
-			if (!this.environmentService.isBuilt || lifecycleService.startupKind === StartupKind.ReloadedWindow || isWeb) {
+			if (!this.environmentService.isBuilt || lifecycleService.startupKind === StartupKind.ReloadedWindow) {
```

Comment:
How is this related?

Reply:
because web didn't support startupKind reloaded window before now, it was hardcoded to always restore files on web. But now it's supported, we can use the dynamic behaviour the electron version has